spec/spec_helper.rb

```diff
@@ -10,6 +10,10 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
 
+# Global variable? Sure, why not? Use $in_travis in your tests to, say, exclude tests from running when on Travis CI.
+# NOTE: Then environment variable being checked here is set in .travis.yml.
+$in_travis = !ENV['TRAVIS'].nil? && ENV['TRAVIS'] == 'true'
+
```

Comment:
meh. Environment variables aren't tweaked so much as simply set during the call to `rspec`, e.g.
```
# emulate travis
TRAVIS=true bundle exec rake spec
# don't emulate travis
bundle exec rake spec
```
i ripped off the `unless: $global_var` pattern from ScholarSphere. Hadn't seen it before and thought it was cool.